index.html

```diff
@@ -35,55 +35,308 @@
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- about section -->
+      <section id="about-section">
+        <div class="about">
+          <h1 class="about-header">About Myself</h1>
+          <h1 class="about-text">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </h1>
+          <p class="connect">LET’S CONNECT</p>
+          <div class="connect-links">
+            <a href="#">
+              <img
+                src="./Assets/Icons/twitter-vector.svg"
+                alt="twitter"
+                class="social-icons"
+              />
+            </a>
+            <a href="#">
+              <img
+                src="./Assets/Icons/linkedin-vector.svg"
+                alt="linkedin"
+                class="social-icons"
+              />
+            </a>
+            <a href="#">
+              <img
+                src="./Assets/Icons/medium-vector.svg"
+                alt="medium"
+                class="social-icons"
+              />
+            </a>
+            <a href="#">
+              <img
+                src="./Assets/Icons/github-vector.svg"
+                alt="gitHub"
+                class="social-icons"
+              />
+            </a>
+            <a href="#">
+              <img
+                src="./Assets/Icons/social-icon-vector.svg"
+                alt="socia;"
+                class="social-icons"
+              />
+            </a>
+          </div>
+          <a href="#" class="project-link">
+            <span class="link">Get my resume</span>
+          </a>
+          <div class="about-languages">
+            <div class="language-header-container">
+              <h3 class="language-header">Lanuages</h3>
+              <img src="./Assets/Icons/icon-down.png" alt="" />
+            </div>
+            <div class="my-language">
+              <div class="my-language-container">
+                <img
+                  src="./Assets/Icons/javascript-icon.png"
+                  alt="javaScript"
+                />
+                <p class="my-language-text">javaScript</p>
+              </div>
+              <div class="my-language-container">
+                <img src="./Assets/Icons/css-icon.png" alt="CSS" />
+                <p class="my-language-text">CSS</p>
+              </div>
+              <div class="my-language-container">
+                <img src="./Assets/Icons/html-icon.png" alt="HTML" />
+                <p class="my-language-text">HTML</p>
+              </div>
+            </div>
+          </div>
+          <div class="language-header-container frameworks">
+            <h3 class="language-header">Frameworks</h3>
+            <img src="/Assets/Icons/icon-right.png" alt="" />
+          </div>
```

Comment:
Because here it's a list, please use ul/li tag to have a good semantic web.

one list for langagues, framework, skills
and inside the first element you have another list javascript/CSS/HTML